go.mod

```diff
@@ -193,12 +194,14 @@ require (
 )
 
 replace (
+	// https://github.com/cometbft/cometbft-db/pull/42
+	github.com/cometbft/cometbft-db => github.com/crypto-org-chain/cometbft-db v0.0.0-20230306031617-b3e4fd3331c4
 	// Ref: https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.10
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.11
 	github.com/crypto-org-chain/cronos/versiondb => ./versiondb
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.7-cronos
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.8-cronos
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/v0.20.7-cronos...v0.20.8-cronos

Reply:
I get a diff on gomod2nix.toml for current v0.20.8-cronos

Reply:
I see, but it should be ok, since it's not used here ;D